Makefile

```diff
@@ -773,6 +773,10 @@ test-integration-test:
 		--reporter $(REPORTER) \
 		--buildapp $(BUILDAPP)
 
+.PHONY: test-gaia-ui-test
+test-gaia-ui-test:
+	./bin/gaia-ui-tests
```

Comment:
I actually wasn't aware of this script as it's not described in mdn at all.

I don't really care about a make target; having a proper script is good enough for me.

(the change to really-clean is welcomed though)